config/default/kustomization.yaml

```diff
@@ -1,12 +1,12 @@
 # Adds namespace to all resources.
-namespace: policy-controller-operator-system
+namespace: policy-controller-operator
 
 # Value of this field is prepended to the
 # names of all resources, e.g. a deployment named
 # "wordpress" becomes "alices-wordpress".
 # Note that it should also match with the prefix (text before '-') of the namespace
 # field above.
-namePrefix: policy-controller-operator-
+namePrefix: policy-op-
```

Comment:
is here something missing or is the dash at the end intended? :)

Reply:
The dash here is correct its a name prefix so this gets added to the name of any resource so for example the name `controller-manager` becomes `policy-op-controller-manager`